### ISSUE TITLE
JS overridding not working

### ISSUE THREAD

**Issue by vitek-rostislav** [shopsys/shopsys#1711]
### What is happening
https://docs.shopsys.com/en/latest/frontend/npm-and-webpack/#i-want-to-override-class-from-shopsysframework-common-package describes how to override a js from the framework, however, when I follow the instructions, I get an error in console:
```
Uncaught TypeError: Class constructor CategoryTreeSorting cannot be invoked without 'new'
    at new ExtendedCategoryTreeSorting (categoryTree.sorting.js:7)
    at Object.init [as callback] (categoryTree.sorting.js:33)
    at Register.registerNewContent (Register.js:71)
    at HTMLDocument.<anonymous> (registerAdmin.js:14)
    at mightThrow (jquery.js:3557)
    at process (jquery.js:3625)
```

Maybe this is an ES6 restriction? https://stackoverflow.com/questions/51860043/javascript-es6-typeerror-class-constructor-client-cannot-be-invoked-without-ne :thinking: 

### Expected result
It is possible to override a JS from the framework following the docs.

**Comment by henzigo** [comment]
Could you post your overridden js file?

**Comment by vitek-rostislav** [comment]
my overridden file:
```
import 'nestedSortable';
import CategoryTreeSorting from 'framework/admin/components/CategoryTreeSorting';
import Register from 'framework/common/utils/Register';

class ExtendedCategoryTreeSorting extends CategoryTreeSorting {
    constructor ($rootTree, $saveButton) {
        super($rootTree, $saveButton);
        this.$rootTree = $rootTree;
        this.protectRoot = $rootTree.hasClass('js-protect-root');

        const _this = this;
        this.$rootTree.nestedSortable({
            listType: 'ul',
            handle: '.js-category-tree-item-handle',
            items: '.js-category-tree-item',
            placeholder: 'js-category-tree-placeholder form-tree__placeholder',
            toleranceElement: '> .js-category-tree-item-line',
            forcePlaceholderSize: true,
            helper: 'clone',
            opacity: 0.6,
            revert: 100,
            change: () => _this.onChange(),
            protectRoot: _this.protectRoot
        });

        $saveButton.click(() => this.onSaveClick());
    }
    static init ($container) {
        const $rootTree = $container.filterAllNodes('#js-category-tree-sorting > .js-category-tree-items');
        const $saveButton = $container.filterAllNodes('#js-category-tree-sorting-save-button');
        if ($rootTree.length > 0 && $saveButton.length > 0) {
            // eslint-disable-next-line no-new
            new ExtendedCategoryTreeSorting($rootTree, $saveButton);
        }
    }
}

(new Register()).replaceCallback('CategoryTreeSorting.init', ExtendedCategoryTreeSorting.init);
```

**Comment by pesektomas** [comment]
Problem is solved in babel (v7.9) - https://github.com/babel/babel/pull/8656. I will prepare PR. For quick fix you have to use `babel/*` in version ^7.9

```
"@babel/core": "^7.9.0",
"@babel/parser": "^7.9.0",
"@babel/preset-env": "^7.9.0",
"@babel/traverse": "^7.9.0",
```
